src/Language/PureScript/Pretty/Values.hs

```diff
@@ -124,7 +124,7 @@ prettyPrintDoNotationElement (DoNotationBind binder val) =
     ]
 prettyPrintDoNotationElement (DoNotationLet ds) =
   concat <$> sequence
-    [ return "let "
+    [ return "let\n"
```

Comment:
So I don't see a good solution to pretty-printing the let without the newline.  Do you have an idea on how to do it?  The code as it is broken since the remainder of the printing of the let assumes a newline has been printed and an indentation has been added.